docs/detections/rules-ui-create.asciidoc

```diff
@@ -286,7 +286,7 @@ alerts.
 +
 NOTE: You can use {kib} saved queries (image:images/saved-query-menu.png[Saved query menu,18,18]) and queries from saved Timelines (*Import query from saved Timeline*) as rule conditions.
 +
-.. Use the *Fields* menu to select a field to check for new terms.
+.. Use the *Fields* menu to select a field to check for new terms. You can also select up to three fields to detect a combination of new terms (for example, a `host.ip` and `host.id` that have never been observed together before).
```

Comment:
There is a limitation currently, when single document has more than 100 unique combination for new terms fields.
https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/new_terms/README.md#limitations-and-future-enhancements
Would be worth to mention it

Reply:
@vitaliidm Thanks for this info, definitely will include it! Do you know what exactly happens if a document falls within this limitation? For example, will the rule still run, or will it fail? Will it still analyze some of the data (such as the first 100 combinations found)? Is there an error or a log message to indicate that some data wasn't analyzed?

Reply:
>  Do you know what exactly happens if a document falls within this limitation? For example, will the rule still run, or will it fail? Will it still analyze some of the data (such as the first 100 combinations found)? Is there an error or a log message to indicate that some data wasn't analyzed?

rule will still run, missing results might happen if number of unique combination too large.
there are no indications whether it happened or not.
In one of previous versions of implementation rule was failing, when number of combination is greater than 100. But we decided to keep current implementation, as it would allow rule to finish run.

At this point there are no workarounds or alternatives to it.
We have slower in performance implementation, that doesn't have this weakness. But has some other [limitations](https://github.com/elastic/kibana/issues/142862#issuecomment-1316865039)

Reply:
@vitaliidm I added the note below to mention the limitation. Please let me know if it needs any adjustment, thanks!

> IMPORTANT: When checking multiple fields, results are limited to no more than 100 unique combinations; any additional combinations will not be detected. If you expect your data to produce more than 100 unique new terms combinations, consider checking other fields to narrow down the search.

Reply:
> IMPORTANT: When checking multiple fields, full results can be produced only for documents with no more than 100 unique combinations of values in these fields. If you expect your data to produce more than 100 unique new terms combinations per document, consider checking other fields to narrow down the search.